.sclng/metadata.toml

```
@@ -0,0 +1,8 @@
+version = "1.0.0"
+owner = "etienne@scalingo.com"
+team = "IST"
+description = "Official Scalingo buildpack - Based on CHH php buildpack, Nginx/PHP-fpm stack"
```

I'm not sure the part about CHH is really interesting in this context, WDYT?

```suggestion
description = "Official Scalingo PHP buildpack"
```

---

You're right. Sorry, I just seen your message.